### ISSUE TITLE
Unorthodox versioning

### ISSUE THREAD

**Issue by blshkv** [drwetter/testssl.sh#1568]
Hi, the latest version was tagged as ```3.0-1```.
I suspect this means ```3.0.1``` really if you normalise it according to common practices:
https://en.wikipedia.org/wiki/Software_versioning

Unusual software versioning causes an additional work for Linux package maintainers like myself. For example, Gentoo portage system does not support ```-``` in the version number, and we have to spend extra time to come up with a fix instead of just coping a previous ebuild file.
Next, it might also cause similar problems with sites like https://repology.org where a software versioning is getting tracked and compared because each distro could name it differently.

Could you stick with just numbers in the version please?

**Comment by drwetter** [comment]
Hmm, I am hestitant.

@ Anyone else?

**Comment by blshkv** [comment]
I think my point is to choose one scheme and stick to it.
It used to be 2.x.y-z, followed by 3.0rc1, and now 3.x-y. These are 3 different schemes.
I guess, it should be 3.0.0-1 if you follow 2.x scheme, but I don't see how ```rc``` can be added here.

Please help to come up with one, so I could implement it within an ebuild.

**Comment by blshkv** [comment]
Just to demonstrate the problem, here is what we have today:
https://repology.org/project/testssl.sh/versions
For the releases ```2.9.5-x```, majority of distros decided to ignore the last number after dash and pushed 2.9.5, but some of them come up with name like 2.9.5_pX or 2.9.5pX
Now, with 3.0-1 release it is same story. It is not clear if ```3.0``` is the version with dropped ```-1``` or a previous version.

**Comment by drwetter** [comment]
TL;DR: will issue another <s>correct</s> appropriate version during the day


I was more thinking on the rpm packaging names e.g. bison-3.5.3-1.3.x86_64

```
prompt% rpm -qi bison-3.5.3-1.3.x86_64
Name        : bison
Version     : 3.5.3
Release     : 1.3
```

In fact it seems that 3.5.3 is the upstream version and -1.3 relates to the distributors version of the package. This is a clear example from the RPM sphere I picked. In the Debian world it seems not that clear.

Thanks for speaking up

**Comment by drwetter** [comment]
done
